pkg/compact/compact.go

```diff
@@ -34,6 +34,7 @@ const (
 )
 
 var blockTooFreshSentinelError = errors.New("Block too fresh")
+var emptyBlockSentinelULID = ulid.MustNew(123, nil)
```

Comment:
This sentinel is for `noMoreCompactionForGroup` I guess? can we name it like this or something similar?

Reply:
OK so I think there is misformation here , I would prefer  something opposite like `noMoreCompactionForGroup` instead of this? What do you think? OR even bool flag return argument to make it more explicit (:

Reply:
I used a separate bool flag as I think that's the clearest what's happening.  The naming was a little awkward but hopefully it makes sense, let me know what you think :)